### PR TITLE
Autobuild docs using Travis CI

### PR DESCRIPTION
**Pull request by liyanhui1228**
Add TravisCI config for pushing docs to gh-pages branch on commits to master.
See: https://docs.travis-ci.com/user/deployment/pages/ for configuration options

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+script:
+  - ./scripts/update_docs.sh
+  - touch docs/.nojekyll
+
+branches:
+  only:
+    - master
+
+deploy:
+  provider: pages
```
Comment (duggelz) [inline .travis.yml]: Can you add a link to https://docs.travis-ci.com/user/deployment/pages/ ?  I had to Google a bit to figure out what this was doing.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  local_dir: docs
+  skip_cleanup: true
+  email: yanhuil@google.com # To satisfy the CLA check, replace this with bot email.
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
```
Comment (chingor13) [inline .travis.yml]: Currently, we're using a personal token with the `public_repo` scope. @rakyll suggested that we may create a bot user for the organization in the future and replace this token with one for the bot.

File: .travis.yml (hunk 1 above)
Comment (liyanhui1228) [reply .travis.yml]: Sorry for not including this link with the PR description. Added it now.

File: .travis.yml (hunk 2 above)
Comment (liyanhui1228) [reply .travis.yml]: Thanks for suggestion!

**Pull request merged**